lib/routes.js

```diff
@@ -24,6 +24,33 @@ module.exports = function(self, options) {
     });
   });
 
+  self.route('post', 'revert', function(req, res) {
+    var id = self.apos.launder.id(req.body.id);
+    return self.revert(req, id, function(err, results) {
+      if (err) {
+        return res.send({
+          status: (typeof (err) === 'string') ? err : 'error'
+        });
+      }
+
+      // On success, grab current version of reverted doc and include url for client-side redirect
+      return self.apos.docs.find(req, { _id: id }).toArray(function(err, docs) {
```

Comment:
This logic does not work because `id` is a commit id, not a doc id. Arrange for your `revert` method to make the resulting draft doc available to its callback so you can do the find with the `_id` of that rather than the commit id.

Reply:
done

Reply:
Good catch, this should be results.id, probably should be singular (result) to indicate that a single doc is returned.

Reply:
done